src/refactorings/extract/extract-variable/parts.ts

```diff
@@ -0,0 +1,33 @@
+import { Code } from "../../../editor/editor";
+import { Selection } from "../../../editor/selection";
+import { Position } from "../../../editor/position";
+
+export { Parts };
+
+class Parts {
```

Comment:
This is the code that should be enriched if we want to handle multi-lines selections.

For now, it gracefully falls back on extracting the whole string instead.